docs/production-deployment/cloud/terraform-provider.mdx

```diff
@@ -110,57 +72,51 @@ set TEMPORAL_CLOUD_API_KEY=<yoursecretkey>
 </TabItem>
 </Tabs>
 
-The `TEMPORAL_CLOUD_API_KEY` is used to authenticate the Terraform provider.
-
-3. (optional) Generate your CA certificate
-
-The CA certificate allows you to authenticate and interact with your Temporal Cloud Namespace.
-You can use an existing CA cert or [create one using tcld](/cloud/certificates).
-Once you have your CA certificate be sure to add the CA `.pem` file to your working directory.
-
-```bash
-mv ca.pem test-temporal-terraform
+Or, pass it in manually in your .tf file using the provider code block
+```yml
+provider "temporalcloud" {
+  api_key = "my-temporalcloud-api-key"
```

Comment:
The API key will be stored in plaintext in the statefile if they use this method. We should either note that, or not recommend it and focus on the env var method.